samples/net/mqtt_publisher/boards/cc3220sf_launchxl.conf

```diff
@@ -0,0 +1,26 @@
+# Networking Config:
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=n
+CONFIG_NET_SOCKETS=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+# Enable SimpleLink WiFi Driver and Socket Offload
+CONFIG_WIFI=y
+CONFIG_WIFI_SIMPLELINK=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
```

Comment:
Wouldn't this fit better into `overlay-tls-offload.conf`?

Reply:
The intent of `overlay-tls-offload.conf` is to enable TLS in the offload case. Currently, offloading is used on the cc3220sf_launchxl regardless of whether TLS is used or not, so these statements belong with the base case, ie. in this file.